### PR TITLE
hover disable fix and spacing variables

### PR DESCRIPTION
**Pull request by peterhass**
see https://github.com/good-hood-gmbh/nebenan-frontend/pull/1007

### CODE REVIEW

File: config/layout.scss [hunk 1]
```diff
@@ -8,3 +8,7 @@ $layout-width: $layout-content-width + $layout-sidebar-width;
 
 $layout-width-m: 660px;
 $layout-width-l: 1280px;
+
+$layout-section-spacing-l: 15px;
```
Comment (peterhass) [inline config/layout.scss]: Don't really like the naming - suggestions?

Comment (max-degterev) [reply config/layout.scss]: $offset-s/m/l

Comment (peterhass) [reply config/layout.scss]: -> offset.scss

Comment (max-degterev) [reply config/layout.scss]: you can use sizes.scss there is $offset-space already

**Review by max-degterev — APPROVED** [review]

**Pull request merged**